src/types/dispute-status-types.js

```diff
@@ -1,3 +1,7 @@
+export const ERROR_TYPES = {
+  ERROR_FETCHING_IPFS: 'Error fetching ipfs content',
```

Comment:
Should this also follow the symbol / string mapping convention used for other errors?